docs/components/servo/_index.md

```diff
@@ -6,18 +6,29 @@ type: "docs"
 description: "A hobby servo is a special type of small motor whose position you can precisely control."
 tags: ["servo", "components"]
 icon: "/components/img/components/servo.svg"
+no_list: true
 # SME: #team-bucket
 ---
 
-The Viam *servo* component supports hobby servos: small motors with built-in closed-loop control.
-Servos can be useful in robotics because their position is easily and precisely controlled.
+{{% alert title="Tip" color="tip" %}}
 
-- The servo component does not support servomotors.
-To configure an industrial servomotor, use the [motor](/components/motor/) component with an [encoder](/components/encoder/).
+The Viam servo component supports [hobby servos](https://learn.adafruit.com/adafruit-motor-selection-guide/rc-servos).
 
-Example wiring diagram for a servo wired to a Raspberry Pi board:
+If you have another type of servomotor, it is probably coupled with an [encoder](/components/encoder), not potentiometer, for position feedback.
+Check your device's data sheet and configure these servos as [encoded motors](/components/motor/gpio/encoded-motor).
 
-![A diagram showing the signal wire of a servo connected to pin 16 on a Raspberry Pi. The servo's power wires are connected to a 4.8V power supply.](../img/servo/servo-wiring.png)
+{{% /alert %}}
+
+A *servo* is a [radio-control (RC) or hobby servo motor](https://learn.adafruit.com/adafruit-motor-selection-guide/rc-servos).
+These are small motors with built-in potentiometer position sensors, enabling you to control the angular position of the servo precisely.
+
+As servos can use a lot of power, drawing voltage away from a [board](/components/board), you should power your servo with its own power supply.
```

Comment:
actually, I always recommend a separate power supply since we have more boards than the pi now, and the current draw from different boards is harder to reason about. It is a cleaner recommendation, even if it is a bit inconvenient.

But if you'd like to explicitly separate the cheaper blue rcs and the beefier gpio servos, that might be the best thing to do here.

Reply:
```suggestion
As servos can use a lot of power, drawing voltage away from a [board](/components/board), you should power your servo with its own power supply in most cases.
```

Could add something like that to allow for the alternative?

Reply:
@npentrel done!